docs/data/material/components/stacked-snackbars/GlobalContentSnackbars.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import SnackbarsProvider, { CloseSnackbarRef } from '@mui/lab/SnackbarsProvider';
+import Button from '@mui/material/Button';
+import useSnackbars from '@mui/lab/useSnackbars';
+import MuiAlert, { AlertProps } from '@mui/material/Alert';
+
+const Alert = React.forwardRef<HTMLDivElement, AlertProps>(function Alert(
+  props,
+  ref,
+) {
+  return <MuiAlert elevation={6} ref={ref} variant="filled" {...props} />;
+});
+
+function MyApp() {
+  const snackbars = useSnackbars();
+
+  return (
+    <React.Fragment>
+      <Button onClick={() => snackbars.showSnackbar()}>
```

Comment:
Would a shorter API work as well?

```suggestion
      <Button onClick={() => snackbars.show()}>
```

What's the need to have "snackbar" in the method?

Reply:
Yes we can make it shorter. 

Some benchmarks:
1. notistack - `const { enqueueSnackbar, closeSnackbar } = useSnackbar();`
2. mantine - `import { showNotification, updateNotification } from '@mantine/notifications';`

So for us, `snackbars.show()`, `snackbars.close()` makes sense after having `const snackbars = useSnackbars()`. I will do the changes.

Reply:
Renamed!